app/uk/gov/hmrc/transitmovementsvalidator/models/MessageTypeJson.scala

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.transitmovementsvalidator.models
+
+sealed trait MessageTypeJson {
```

Comment:
Should we just have one `MessageType` object that holds both XML and Json schema information? We have all the Json schema, so I don't see a reason to separate them.